packaging/skia-perf-regression-copy-surface-as-draw.patch

```diff
@@ -0,0 +1,1199 @@
+diff --git a/src/gpu/GrDrawTarget.cpp b/src/gpu/GrDrawTarget.cpp
```

Comment:
Can you include the commit message before the patch? Right now it is totally unclear that it comes from upstream, what the CL number is etc.
Plus you can also mention that this is only necessary for M44.